lib/puppet/feature/vendors_dsc.rb

```diff
@@ -2,10 +2,11 @@
 require Pathname.new(__FILE__).dirname + '../../' + 'puppet_x/puppetlabs/dsc_symlink'
 
 Puppet.features.add(:vendors_dsc) do
-  if Puppet::Util::Platform.windows?
-    DscSymlink.set_symlink_folder
-    true
-  else
-    false
-  end
+  # if Puppet::Util::Platform.windows?
+  #   DscSymlink.set_symlink_folder
+  #   true
+  # else
+  #   false
+  # end
```

Comment:
How should I disable the symlink? Removing this file requires removing the entire test suite. Returning false works but seems silly.